sdks/python/container/py2/build.gradle

```diff
@@ -66,7 +66,8 @@ docker {
                   project.rootProject["docker-tag"] : project.sdk_version)
   files "../Dockerfile", "./build"
   buildArgs(['py_version': "2.7",
-             'pull_licenses': project.rootProject.hasProperty(["docker-pull-licenses"])])
+             'pull_licenses': project.rootProject.hasProperty(["docker-pull-licenses"]) ||
+                     project.rootProject.hasProperty(["isRelease"])])
```

Comment:
Currently, the `isRelease` tag needs to be passed with commands. Ideally, this tag should be set by checking branch name.

Adding this support to docker images are not that helpful for now, but could be used when we support auto setting `isRelease` tag.

Reply:
I see, thanks. I did not know that we need to set isRelease manually.